include/zephyr/drivers/fuel_gauge.h

```diff
@@ -63,9 +63,21 @@ extern "C" {
 #define FUEL_GAUGE_TEMPERATURE		FUEL_GAUGE_STATE_OF_CHARGE + 1
 /** Battery voltage (uV) */
 #define FUEL_GAUGE_VOLTAGE		FUEL_GAUGE_TEMPERATURE + 1
+/** Battery Mode (flags) */
+#define FUEL_GAUGE_MODE			FUEL_GAUGE_VOLTAGE + 1
```

Comment:
`FUEL_GAUGE_MODE` seems pretty specific to SBS so I'd prefer if we add the SBS bit here. I've tried to separate SBS properties from those are more generic.

Personally, I'd prefer all of these to prefixed with SBS. Is that amenable to you?

Reply:
I think that makes sense. Since properties like "max peak power", "sustained peak power", "max peak current" etc are not defined in SBS, these can simply be defined without the "SBS_" prefix. I believe we should do it as part of a separate PR since that would end up modifying all existing properties defined until now.

Reply:
@rgundi:

I believe I misunderstood your prior comment. I think we can keep the older properties as they are though. I really just meant these new properties should have the `SBS_` prefix. I think properties like `remaining_capacity` don't have to be specific to SBS. However, I'm still okay with changing everything to prefix `SBS_` if that's your intention. That's certainly what inspired these properties but I was hopeful many of them could be useful for other more generic fuel gauge ICs as well.

Reply:
Sure. Though I feel it is better to have the SBS_ prefix to indicate it is a SBS defined property to distinguish it from a non-SBS property. If there's a generic FG making use of an SBS property there should be no harm if it sees the prefix SBS_. Also, the new properties that I added are very much needed for a battery so i am not sure how it differs from the old properties which are already present.

Reply:
I like your idea. Let's go for it. I've approved and will approve your next PR adding the SBS_ prefix. Just wanted to indicate what my previous mental model was, but I like your idea. Let's do it.